core/module/used_refinements_spec.rb

```diff
@@ -0,0 +1,36 @@
+require_relative '../../spec_helper'
+
+describe "Module.used_refinements" do
+  ruby_version_is "3.2" do
+    it "returns list of all refinements imported (or activated/active) in the current scope" do
```

Comment:
Heh, sorry for ambiguity. It was kind of suggestion - I don't know which term is more correct (imported, activated or active).

The term _import_ is used in the documentation, so probably it's more preferred:
```
[using(module) → self](https://rubyapi.org/3.1/o/module#method-i-using)

Import class refinements from module into the current class or module definition.
```



```suggestion
    it "returns list of all refinements imported in the current scope" do
```